Epsilon.Host.Frontend/.env.example

```diff
@@ -1,2 +1,3 @@
 VITE_USER_ID=
-VITE_EPSILON_API_ENDPOINT=
\ No newline at end of file
+VITE_EPSILON_API_ENDPOINT=
+VITE_IN_DEVELOPMENT=
```

Comment:
Vite has environments built in, please use those instead: https://vitejs.dev/guide/env-and-mode.html